<?xml version="1.0"?>
<launch>
    <arg name="pure_pursuit_config" default="$(find speed_daemons_pure_pursuit)/config/pure_pursuit.yaml" />
    <rosparam file="$(arg pure_pursuit_config)" command="load"/>
    <node pkg="speed_daemons_pure_pursuit" name="visualize_markers_node" type="visualize_markers.py" output="screen"/>
    <node pkg="speed_daemons_pure_pursuit" name="pure_pursuit_visualizer" type="visualize.py" output="screen"/>
    <node pkg="speed_daemons_pure_pursuit" name="pure_pursuit_node" type="pure_pursuit.py"  output="screen">
        <param name="waypoints_filepath" value="$(find speed_daemons_pure_pursuit)/waypoints/kelley_waypoints.csv"/>
    </node>
    <node pkg="speed_daemons_pure_pursuit" name="dead_mans_switch" type="dead_mans_switch.py" output="screen"/>
	

</launch>
